js/adapt-contrib-tutor.js

```diff
@@ -10,7 +10,19 @@ define([
         };
 
         if (view.model.has('_isCorrect')) {
-            alertObject._classes = view.model.get('_isCorrect') ? 'correct' : 'incorrect';
+            // Attach specific classes so that feedback can be styled.
+            if (view.model.get('_isCorrect')) {
+                alertObject._classes = 'correct';
+            } else {
+                if (view.model.has('_isAtLeastOneCorrectSelection')) {
+                    // Partially correct feedback is an option.
+                    alertObject._classes = view.model.has('_isAtLeastOneCorrectSelection')
```

Comment:
is this not doing the same thing as the `if' statement on the line above? Did you maybe mean `get` rather than `has` here?